source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+    **Note:** The Mattermost mobile apps are signed and have certificates associated with Mattermost and the public app stores. This means they will not work if you are privately hosting the Mattermost Push Proxy Service.
+
+`Mattermost for Android Devices <https://play.google.com/store/apps/details?id=com.mattermost.rn&hl=en_US>`_ (via Google Play)
+`Mattermost of iOS Devices <https://apps.apple.com/us/app/mattermost/id1257222717>`_ (via Apple App Store)
+
+If you do not desire (or require) additional security provide via an EMM Provider, your deployment is complete. Feel free to point your users to the `available documentation <https://docs.mattermost.com/guides/user.html>`_.
+
+The sub-sections below serve as a high-level guide to understanding this deployment model. When necessary, we point to the documentation for technical instruction.
+
+    **Note:** At a minimum, we recommend you follow our recommended steps to secure your deployment.
+
+********************************************
+Using an EMM Provider with Public Store Apps
+********************************************
+
+EMM providers help extend security parameters to the Mattermost mobile applications. The AppConfig standard makes this possible. `Review the Mattermost AppConfig Values Documentation <https://docs.mattermost.com/mobile/mobile-appconfig.html#mattermost-appconfig-values>`_ for a complete list of available parameters.  
+
+When going this route, you should consider:
+
+- What is the mobile policy, is it company-owned or BYOD?
+- Do you know what devices will be used if customer-owned?
+- What OS do you want to start with in testing?
+- Creating a sample configuration then run validation tests against each config item
+
+**********************************
+Configuring Mattermost to Use HPNS
+**********************************
+
+Configuring your Mattermost Server to use the Mattermost HPNS is a single configuration item. This is covered in our `Hosted Push Notification Service documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_. 
+
+Next your users would need to install the mobile application on their device. If desired, you can further configure security capabilities using an EMM provider.
+
+**********************************
+Updating via the Public Store Apps
+**********************************
+
+While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost mobile applications are available. Simultaneously, you should check any compatibility requirements for the mobile apps and the Mattermost server. 
+
+It is often easier to upgrade the mobile apps. However, not all provided updates are compatible with all previous versions of the Mattermost server. 
+
+    **Note:** Only updating the mobile apps, or updating the mobile apps before the Mattermost server, can result in incompatibility.
+
+----
```

Comment:
Could we re-word this to provide clearer guidance on what to do? I am not sure of the steps but if we are saying there is a risk here, are we able to provide simple guidance to mitigate it? Or is this more complex? e.g.
We recommend that you: 
Check the compatibiltiy requirements
Validate versions connecting to server
Update server
Update apps

Reply:
+1 I think it’s reasonable to add this and links to the pages that show the compatibility and version notes.